Assets/Scripts/UI/UI.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using PlayerInput;
+
+public class UI : MonoBehaviour
+{
+    public Image AbilityCoolDown;
+    public Image Ability;
+    public Image PreviousAbility;
+    public Image Grapple;
+    public Image GrappleCooldown;
+    public Sprite GreenAbilitySprite;
+    public Sprite RedAbilitySprite;
+    public Sprite YellowAbilitySprite;
```

Comment:
Is there a reason why these aren't part of the sprite sheet but the other non ability colors are?

Reply:
are you talking about where they are stored in the Resources/Sprites folder?

Reply:
Like half of them are individual sprites (the ability ones) and half of them are children of the Sprite sheet (the solid colors).

Reply:
oh i think its because originally i got the sprites fro the abilities individually a couple days before i got the sprite sheet so i had already made sprites based from the individual PNGs that i got and for the solid colors  got from the sprite sheet by slicing them out